cli/build.rs

```diff
@@ -70,13 +70,13 @@ fn main() {
   let bundle_path = o.join("COMPILER_SNAPSHOT.js");
   let snapshot_path = o.join("COMPILER_SNAPSHOT.bin");
   let mut custom_libs: HashMap<String, PathBuf> = HashMap::new();
+  custom_libs
+    .insert("lib.window.d.ts".to_string(), c.join("js/lib.window.d.ts"));
```

Comment:
I am not 100% on this name... it implies it is really Deno's cherry picking of the global scope/dom.  I think it should still carry Deno in it somehow... like maybe `lib.deno_window.d.ts`?

Reply:
Agree, `lib.deno_window.d.ts` SGTM